### ISSUE TITLE
Improve the explanation for the "Layout" setting.

### ISSUE THREAD

**Issue by costdev** [WordPress/gutenberg#40861]
Originally reported on [Trac 55670](https://core.trac.wordpress.org/ticket/55670).
## What problem does this address?

In 6.0 there's is this text about the new layout setting:
> Customize the width for all elements that are assigned to the center or wide columns.

That is a really confusing text. I would avoid the wording of "columns" cause that conflicts when users are inside a column block.

I do found this layout setting a complex setting; as it only has effect on "internal" blocks; and the Wide setting has only effect on those block that are able to set Wide like an image, but not a paragraph block.

And more: when users our inside a Column of a Column block, they also have a Width setting of that particular Column. That's even more confusing, right.

I do wonder if WordPress would still needs such a profound setting like this. And have more the basic settings.

## What is your proposed solution?
But if this continious, which probably will. Please do change the text; for example:

> Customize the width for all blocks inside that layout element. The Content value sets the general width of every block. The Wide value sets the width of every block that is able to have a wide alignment.

**Comment by annezazu** [comment]
Related issue here: https://github.com/WordPress/gutenberg/issues/36082 

Might be best to default to that one above? Don't feel strongly either way - just know it's been a big area of feedback.

**Comment by maddisondesigns** [comment]
I agree with @costdev. I'm trying to make some theme adjustments and I have no idea what these fields actually do and the description text makes no sense. I can see that the `Content` field changes the width of the blocks contained within the Group block, but have absolutely no idea what the `Wide` field does. My first thought was that it somehow affects the Group block when it's set to `Alignwide`, but that doesn't seem to be the case.
